draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1333,13 +1335,14 @@ than the previously generated UUID. If this is not the case then any number
 of things could have occurred, such as clock rollbacks,
 leap second handling, and counter rollovers. Applications SHOULD embed sufficient
 logic to catch these scenarios and correct the problem to ensure that the next
-UUID generated is greater than the previous. To handle this scenario, the
+UUID generated is greater than the previous, or at least report an appropriate error.
```

Comment:
I think it's helpful to indicate that if the implementation gets into some sort of pickle where it cannot return a unique ID, that returning an error instead is perhaps the right behavior.